columnq/src/columnq.rs

```diff
@@ -50,6 +51,14 @@ impl ObjectStoreProvider for ColumnQObjectStoreProvider {
                 }
 
             },
+            "az" => {
```

Comment:
TODO:
1. use `io::BlobStoreType`
2. adds more schema for Azure Storage, including `adl://`, `abfs://`, `abfss://`, `azure://`

Reply:
done in b65c246659d9c9e054f6f5d3ffa199016acfbb87